src/Avalonia.Controls/AutoCompleteBox/AutoCompleteBox.Avalonia.cs

```diff
@@ -0,0 +1,22 @@
+﻿// This file contains improvements to AutoCompleteBox Avalonia respect the upstream Microsoft AutoCompleteBox
+// These improvements are added to a separate file to try not to interfere when you eventually need to sync Avalonia AutoCompleteBox with upstream.
```

Comment:
I don't think it's in sync now? Shouldn't be a problem to add a single property with no other lines modified.

Reply:
Not knowing the current status, I thought not to disturb any synchronization so that the changes upstream are evident. Do I merge?

Reply:
Let's just move these to AutoCompleteBox.Properties.cs